.ansible-lint

```diff
@@ -3,3 +3,4 @@ profile: production
 warn_list:
   - role-name  # until role name is fixed on galaxy https://github.com/riemers/ansible-gitlab-runner/pull/312
   - no-changed-when  # TODO in future
+  - yaml[line-length]
```

Comment:
I'm not a fan of this. Instead we should try to fix the actual lint violations in a clean manner.